backend/src/index.js

```diff
@@ -271,8 +303,11 @@ async function main() {
           "node-stats",
           {
             validatorAmount: validators.length,
+            seatPriceAmount: seatPrice,
             onlineNodeAmount: onlineNodes.length,
             proposalAmount: proposals.length,
+            totalStakeAmount: totalStake,
+            epochStartHeightAmount: epochStartHeight,
```

Comment:
"Amount" prefix does not make sense here:

```suggestion
            epochStartHeight,
```